runtime/v2/manager.go

```diff
@@ -140,14 +141,73 @@ func (m *MM) ValidateGenesis(genesisData map[string]json.RawMessage) error {
 	return nil
 }
 
-// InitGenesis performs init genesis functionality for modules.
-func (m *MM) InitGenesis() {
-	panic("implement me")
+// ExportGenesis performs export genesis functionality for modules
+func (m *MM) ExportGenesis(ctx context.Context) (map[string]json.RawMessage, error) {
+	return m.ExportGenesisForModules(ctx, []string{})
 }
 
-// ExportGenesis performs export genesis functionality for modules
-func (m *MM) ExportGenesis() {
-	panic("implement me")
+// ExportGenesisForModules performs export genesis functionality for modules
+func (m *MM) ExportGenesisForModules(ctx context.Context, modulesToExport []string) (map[string]json.RawMessage, error) {
```

Comment:
personal preference:
```suggestion
func (m *MM) ExportGenesisForModules(ctx context.Context, modulesToExport ...string) (map[string]json.RawMessage, error) {
```